From cd583da3ff1a35d8cbfa12e7f915b006ad16fd58 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 28 Apr 2015 20:32:34 +0800
Subject: [PATCH] package_gui.py: add ldd/gzip/iputils to default installation

While partition rootfs as lvm, we need to invoke build_lvm_initramfs.sh
to generate initramfs. It requires the packages ldd, gzip and iputils to
be installed to target image.

So while customize to select packages, we need to have them to make
sure the target could boot even though the user does not select them.

Upstream-Status: Inappropriate [WRLinux specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/iw/package_gui.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/iw/package_gui.py b/pyanaconda/iw/package_gui.py
index 7a66cd9..a1471f5 100644
--- a/pyanaconda/iw/package_gui.py
+++ b/pyanaconda/iw/package_gui.py
@@ -42,7 +42,7 @@ class PackageSelectionWindow (InstallWindow):
         self.backend.resetGroup()
 
         # The extra packages make sure the target could boot
-        for pkg in ["packagegroup-wr-base", "packagegroup-wr-boot"]:
+        for pkg in ["packagegroup-wr-base", "packagegroup-wr-boot", "ldd", "gzip", "iputils"]:
             if self.pkgsel.packages.get_available_package(pkg):
                 self.backend.selectPackage(pkg)
 
-- 
1.9.1

